### PR TITLE
OCM-11730 | ci: fix id:75534

### PR DESCRIPTION
**Pull request by aaraj7**
https://privatebin.corp.redhat.com/?075691d0459cf459#ARZVyHCrJVsvDqif7B5dMT7m4UnrDyHuFGj9Kw1AR88K

### CODE REVIEW

File: tests/e2e/test_rosacli_cluster.go [hunk 1]
```diff
@@ -3554,24 +3554,23 @@ var _ = Describe("HCP cluster creation supplemental testing",
 				flags, err := profilehandler.GenerateClusterCreateFlags(customProfile, rosaClient)
 				Expect(err).ToNot(HaveOccurred())
 
-				command := "rosa create cluster --cluster-name " + clusterName + " " + strings.Join(flags, " ")
+				command := "rosa create cluster --cluster-name " + clusterName + " " + strings.Join(flags, " ") + " " + "--mode auto -y" //nolint
```
Comment (xueli181114) [inline tests/e2e/test_rosacli_cluster.go:3557]: Why not wrap to two lines?

Comment (aaraj7) [reply tests/e2e/test_rosacli_cluster.go]: fixed it

**Pull request merged**